js/client-chat-tournament.js

```diff
@@ -277,6 +277,7 @@
 						if (!this.info.isJoined)
 							this.toggleBoxVisibility(false);
 						this.room.$chat.append("<div class=\"notice tournament-message-start\">The tournament has started!</div>");
+						if (this.info.format.substr(0, 4) === 'gen5' && !Tools.loadedSpriteData['bw']) Tools.loadSpriteData('bw');
 						break;
 
```

Comment:
I guess there's two ways to go from here:
1. Wait on any of the earlier specified events and load the sprite data before they begin a battle (like the pullreq now, as well as loading it for the joined players in a tour), and the load the data on join for spectators.
2. Only load sprites when you join a gen 5 battle, even if you're a player, and accept that the team preview is pretty ugly. 

Neither one is perfect, but for the players either method would work.